### PR TITLE
hide the config global-kill

### PR DESCRIPTION
**Pull request by XuHuaiyu**
The feature is not an experimental feature, thus we need to hide the config description.

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/tidb/pull/23395

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: explain-overview.md [hunk 1]
```diff
@@ -322,25 +321,6 @@ mysql> EXPLAIN SELECT /*+ INL_HASH_JOIN(t1, t2) */ * FROM t1, t2 WHERE t1.id = t
 6 rows in set (0.00 sec)
 ```
 
-#### Index Merge Join 示例
```
Comment (lzmhhh123) [inline explain-overview.md:325]: Also hide it in `explain-subqueries.md`

Comment (lzmhhh123) [reply explain-overview.md:325]: Also in `optimizer-hints.md`

**Review by lzmhhh123 — APPROVED** [review]

**Comment by Joyinqin** [comment]
/remove-translation/doing

**Comment by Joyinqin** [comment]
/remove-translation/doing

**Pull request merged**